sys/schedstatistics/schedstatistics.c

```diff
@@ -31,15 +31,18 @@ void sched_statistics_cb(kernel_pid_t active_thread, kernel_pid_t next_thread)
 {
     uint32_t now = xtimer_now().ticks32;
 
-    /* Update active thread runtime, there is always an active thread since
-       first sched_run happens when main_trampoline gets scheduled */
-    schedstat_t *active_stat = &sched_pidlist[active_thread];
-    active_stat->runtime_ticks += now - active_stat->laststart;
+    /* Update active thread stats */
+    if (active_thread != KERNEL_PID_UNDEF) {
```

Comment:
Should it also be documented in the SCHED_CB docs that `KERNEL_PID_UNDEF` is possible now?

Reply:
there's not much documentation for sched_cb I think, other than that the whole module does what it does. where would you like to see extra documentation?

Reply:
Maybe leave this to a follow-up, there is indeed not much documentation, better do this right in a new PR, I can open something later this week I think

Reply:
See #14334